FSJ_django20_project/FSJ/templates/FSJ/coord_awards_list.html

```diff
@@ -106,7 +112,8 @@
                                     {{year}} <br>
                                 {% endfor %}
                             </td>
-                            <td>{{award.deadline}}</td>
+                            <td>{{award.start_date}}</td>
```

Comment:
In progress awards are missing start and end date in the template, also all table headings (new, in progress, submitted) need to be changed to colspan = 9